backends-velox/src/test/scala/io/glutenproject/execution/VeloxParquetDataTypeValidationSuite.scala

```diff
@@ -445,16 +445,20 @@ class VeloxParquetDataTypeValidationSuite extends VeloxWholeStageTransformerSuit
     }
   }
 
-  ignore("Velox Parquet Write") {
+  test("Velox Parquet Write") {
     withSQLConf(("spark.gluten.sql.native.writer.enabled", "true")) {
       withTempDir {
         dir =>
           val write_path = dir.toURI.getPath
           val data_path = getClass.getResource("/").getPath + "/data-type-validation-data/type1"
-          val df = spark.read.format("parquet").load(data_path)
+          // Spark 3.4 native write doesn't support Timestamp type.
+          val df = spark.read.format("parquet").load(data_path).drop("timestamp")
```

Comment:
Shall we check for version here also and only drop timestamp column when version is 3.4 or above?

Reply:
We do not support timestamp for all Spark version for now. The Spark version constraint is due to the `WriteFilesExec`.

Reply:
@ayushi-agarwal Spark 3.3 and 3.2 also do not support Timestamp, and we didn't test it properly due to the lack of result validation code.